test/nn/test_pack_linear.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python3
+# Copyright 2024 Arm Limited and/or its affiliates <open-source-office@arm.com>
```

Comment:
Nit: we usually don't do top level copyrights, as this one is already covered by 
https://github.com/pytorch/pytorch/blob/001ad5bef521aa92ae8209279fa549e3f46c0ecb/LICENSE#L39-L40